apps/judicial-system/web/src/routes/Court/Indictments/CourtRecord/CourtRecord.tsx

```diff
@@ -38,21 +40,21 @@ import * as constants from '@island.is/judicial-system/consts'
 import { courtRecord as m } from './CourtRecord.strings'
 
 const CourtRecord: React.FC = () => {
-  const { workingCase, isLoadingWorkingCase, caseNotFound } = useContext(
-    FormContext,
-  )
+  const {
+    workingCase,
+    setWorkingCase,
+    isLoadingWorkingCase,
+    caseNotFound,
+  } = useContext(FormContext)
   const [navigateTo, setNavigateTo] = useState<keyof stepValidationsType>()
 
   const { formatMessage } = useIntl()
   const { transitionCase } = useCase()
 
-  const {
-    files,
-    handleS3Upload,
-    handleRemoveFromS3,
-    handleRetry,
-    allFilesUploaded,
-  } = useS3Upload(workingCase)
+  const { files, handleS3Upload, handleRetry, allFilesUploaded } = useS3Upload(
```

Comment:
Dose the functionality break if we use the `useS3UploadV2` here?

Reply:
UseS3UploadV2 is not a drop-in replacement for useS3Upload. You need some extra code for the upload functionality to work if we use v2 but the remove functionality does not break. This PR is just about stop using the remove functionality from the old hook.

Reply:
NB. this pr is one step in a bigger story that is to completely remove useS3Upload from our codebase and use the newer useS3UploadV2 instead. I'm just cutting that story down into smaller pieces.

Reply:
I see, sounds good.